app/scripts/views/mixins/floating-placeholder-mixin.js

```diff
@@ -30,17 +32,26 @@ define(function (require, exports, module) {
      *        should be shown.
      */
     showFloatingPlaceholder: function (inputEl) {
-      var $inputEl = $(inputEl);
-      var placeholder = $inputEl.attr('placeholder');
+      var placeholder = this.$el.find(inputEl).attr('placeholder');
 
       // If the placeholder for the element was already converted, no
       // further conversion will occur.
-      if (placeholder !== '') {
-        $inputEl.removeAttr('placeholder');
-        $inputEl.prev('.label-helper').text(placeholder).animate( {'top': '-17px'}, 400);
+      // The check for existence is because of the strict equality check,
+      // if placeholder is undefined, then it should not go into the block
+      if (placeholder && placeholder !== '') {
```

Comment:
> if placeholder is undefined, then it should not go into the block

If the placeholder is undefined, then it *will* go into the block, the strict equality check matches against the empty string only.